### DISCUSSION TITLE
Support for max_loaded_maps and num_parallel variables/parameters

### DISCUSSION THREAD

**Discussion by jars101** [intel-analytics/BigDL#11138] [General]
It does not seem that ollama running on ipex-llm supports the most recent max_loaded_maps and num_parallel variables/parameters. Is it supported in current ollama version under llama-cpp? how does one enables it? thanks

**Comment by rnwang04** [comment]
Hi @jars101 , would you mind raising this to [Issues](https://github.com/intel-analytics/ipex-llm/issues) instead of Discussions here : )

**Reply by jars101** [reply]
hello, sure- done.
